src/main/java/frc/robot/subsystems/PoseEstimatorSubsystem/PhotonVisionSystem.java

```diff
@@ -51,4 +54,21 @@ public Optional<EstimatedRobotPose> getEstimatedGlobalPose(Pose2d prevEstimatedR
 
   // can add other functions here later to check for specific april tags I think
 
+  // this one might get the angle up to april tag ID 7 (middle of blue speaker) but idk
+  public Optional<Double> GetYawToSpeaker() {
```

Comment:
Does this get yaw or pitch? Need to be offset anyway because the speaker hole is above the apriltag.

Reply:
> Does this get yaw or pitch? Need to be offset anyway because the speaker hole is above the apriltag.

It gets pitch oops. I'll fix it at the meeting tn